src/main/scala/org/hatdex/hat/api/json/DataStatsFormat.scala

```diff
@@ -16,57 +16,74 @@ import play.api.libs.json._
 import scala.collection.immutable.HashMap
 
 trait DataStatsFormat extends DataDebitFormats {
-  implicit val mapReads: Reads[HashMap[String, Long]] = new Reads[HashMap[String, Long]] {
-    def reads(jv: JsValue): JsResult[HashMap[String, Long]] = {
-      val fields: Seq[(String, Long)] = jv.as[JsObject].fields.map {
-        case (k, v) =>
-          k -> v.as[Long]
+  implicit protected val dataCreditStatsFormat: Format[DataCreditStats]       = Json.format[DataCreditStats]
+  implicit protected val dataStorageStatsFormat: Format[DataStorageStats]     = Json.format[DataStorageStats]
+  implicit protected val inboundDataStatsFormat: Format[InboundDataStats]     = Json.format[InboundDataStats]
+  implicit protected val outboundDataStatsFormat: Format[OutboundDataStats]   = Json.format[OutboundDataStats]
+  implicit protected val richDataDebitFormat: Format[RichDataDebit]                                  = RichDataJsonFormats.richDataDebitFormat
```

Comment:
Can you try to re run the `sbt scalafmt` please? This `=` is in a very odd position